HelpSource/Guides/WritingHelp.schelp

```diff
@@ -338,4 +306,35 @@ The easiest way to contribute to the documentation is:
     code::
     git push origin -u doc_updates
     ::
-    5. Submit your pull request through github, from your branch doc_updates to SuperCollider develop
+    5. Submit your pull request through github, from your branch doc_updates to SuperCollider main
```

Comment:
PRs are generally pointing to `develop`, or possibly `3.x`, but never to `main`. I'd leave `develop` here.

Reply:
fixed this in latest commit